hooks/{}

```diff
@@ -0,0 +1,7 @@
+HTTP/1.1 200 OK

```

Comment:
I'm not sure why this keeps popping up, one of our redirections isn't working correctly.. looking..